src/BenchmarkDotNet.Core/Configs/ReadOnlyConfig.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+
+using BenchmarkDotNet.Analysers;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Exporters;
+using BenchmarkDotNet.Filters;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Order;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Validators;
+
+using JetBrains.Annotations;
+
+namespace BenchmarkDotNet.Configs
+{
+    [PublicAPI]
+    public class ReadOnlyConfig : IConfig
+    {
+        #region Fields & .ctor
```

Comment:
what is the point of adding regions to such simple class?

Reply:
Will fix:)